eventsourcing/src/main/java/org/axonframework/eventsourcing/eventstore/DefaultEventStoreTransaction.java

```diff
@@ -26,6 +26,7 @@
 import java.util.Objects;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.atomic.AtomicBoolean;
```

Comment:
This file is needed for fixing the bug in https://github.com/AxonFramework/AxonFramework/pull/3302.
If not, testcases don't run. 

Will remove this file once this is merged